source/index.txt

```diff
@@ -1,159 +1,153 @@
-     - Compass Readonly
-     - Compass Isolated
+            :ref:`To learn more, see Query Your Data<query-bar-filter>`
 
-   * - Interact with :ref:`documents <compass-documents>`,
-       :ref:`collections <collection-tab>`, and
-       :ref:`databases <database-tab>` with full CRUD functionality
-     - |checkmark|
-     - 
-     - |checkmark|
+      .. image:: /images/tabs/query-your-data.png
+         :alt: MongoDB Compass collection query image
 
-   * - Create and execute :ref:`queries <compass-query-bar>` and
-       :ref:`aggregation pipelines <compass-agg-builder>`
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
+   .. tab:: Create aggregation pipelines
+      :tabid: create
 
-   * - Create and delete :ref:`indexes <compass-indexes>`
-     - |checkmark|
-     - 
-     - |checkmark|
+      .. procedure::
 
-   * - View and optimize query performance with visual
-       :ref:`explain plans <explain-plans>`
-     - |checkmark|
-     - 
-     - |checkmark|
+         .. step:: Insert documents into your collections
 
-   * - Kerberos, LDAP, and x.509 authentication
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
+            Insert documents into your collections in two ways, JSON Mode and a Field-by-Field Editor.
 
-   * - :doc:`Schema Analysis </schema>`
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
+            :ref:`To learn more, see Insert Documents<insert-documents>`
 
-   * - :doc:`Real Time Server Stats </performance>`
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
+         .. step:: Create aggregation pipelines
 
-   * - Create, delete and edit :ref:`document validation <validation>`
-       rules
-     - |checkmark|
-     - 
-     - |checkmark|
-
-   * - Error collection and crash reporting
-     - |checkmark|
-     - |checkmark|
-     -
+            Write aggregation pipelines that allow documents in a collection or view to pass through multiple stages where they are processed into a set of aggregated results.
 
+            :ref:`To learn more, see Aggregation Pipeline Builder<compass-agg-builder>`
+
+      .. image:: /images/tabs/create-aggregation-pipelines.png
+         :alt: MongoDB Compass aggregation pipeline image
+
+   .. tab:: Run commands in the shell
+      :tabid: run
+
+      .. procedure::
+
+         .. step:: Connect to your deployment
+
+            Connect to a MongoDB deployment hosted on MongoDB Atlas, or a deployment hosted locally on your own machine.
+
+	    :ref:`To learn more, see Connect to MongoDB<connect-run-compass>`
+
+         .. step:: Work with your data in the MongoDB shell
+
+            Use the embedded MongoDB Shell in Compass to control your data in an interactive JavaScript environment.
+
+            :ref:`To learn more, see Embedded MongoDB Shell<embedded-mongodb-shell>`
+
+      .. image:: /images/tabs/run-commands-in-the-shell.png
+         :alt: MongoDB Compass embedded shell image
+
+.. kicker:: Related Products & Resources
```

Comment:
[question]

It's very rare for us to use an ampersand in the documentation. Just curious if this is an intentional design choice. Is there any concern about this making things a touch more difficult for non-native English speakers who might not be as familiar with this symbol?

Reply:
Perhaps better addressed by @allisonmui9! Pinging...

Reply:
@jeff-allen-mongo not particularly intentional. Feel free to change to 'Related Products and Resources.'